performance/tests/Marquee.test.js

```diff
@@ -54,10 +54,10 @@ describe('Marquee', () => {
 	});
 
 	describe('Multiple Marquees', () => {
-		it('mounts', async () => {
-			const counts = [10, 100, 1000];
-			for (let index = 0; index < counts.length; index++) {
-				const count = counts[index];
+		const counts = [10, 40, 70, 100];
+		for (let index = 0; index < counts.length; index++) {
```

Comment:
This still runs tests sequentially, so the overall time spent running test is about the same. But this allows us to generate test cases per sample entry.